utils/bluez-tools/Makefile

```diff
@@ -38,7 +38,8 @@ endef
 define Package/bluez-tools/install
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/bt-agent.init $(1)/etc/init.d/bt-agent
-	$(CP) ./files/bt-agent.cfg $(1)/etc/bt-agent.cfg
+	$(INSTALL_DIR) $(1)/etc/config
```

Comment:
You also need to place the uci config in `/etc/config `without the `cfg` extension.
Also add a `define Package/bluez-tools/conffiles` definition to your makefile https://github.com/openwrt/packages/search?q=conffiles, so this gets save during sysupgrade.

Reply:
done